adlfs/spec.py

```diff
@@ -235,6 +240,7 @@ class AzureBlobFileSystem(AsyncFileSystem):
     def __init__(
         self,
         account_name: str = None,
+        account_host: str = None,
```

Comment:
It's unfortunate, but we'll need to put this new parameter at the end. That way we don't break users using positional args `AzureBlobFileSystem(account_name, key, ...)`

Reply:
I made it the last parameter so it doesn't break anything positional.